docs/STATE-CONFIGS.md

```diff
@@ -0,0 +1,194 @@
+# Reggie state configs
+
+State configuration files are used to describe the state as well as how we ingest the raw voter data and turn it into more usable data.
+
+## Location
+
+State config files are found in the `reggie/configs/data/` directory and are in YAML format.
+
+## Required fields
+
+The following fields are required in the config for Reggie:
+
+* ??
+
+The following fields are required in the config for VoteShield:
+
+* `voter_status`
+* `birthday_identifier`
+* `party_identifier`
+* ??
+
+## State config
+
+### State basics
+
+Values that describe basics about the state itself.
+
+| Field          | Type   | Values | Description                                       | Example    |
+| -------------- | ------ | ------ | ------------------------------------------------- | ---------- |
+| `state`        | string |        | The ID of the state.                              | `new_york` |
+| `abbreviation` | string |        | The two-letter, lowercased abbreviation of state. | `ny`       |
+
+
+## Vote file and history package
+
+### File basics
+
+| Field                                 | Type    | Values       | Description                                                                | Example |
+| ------------------------------------- | ------- | ------------ | -------------------------------------------------------------------------- | ------- |
+| `format.segmented_files`              | boolean | `true|false` | Affects download ???                                                       | `true`  |
+| `format.separate_hist`                | boolean | `true|false` | History is a separate file from the voter file.  (No reference in code???) | `true`  |
+| `format.separate_counties_voter_file` | boolean | `true|false` | (No reference in code???)                                                  | `true`  |
+| `format.separate_counties_hist_file`  | boolean | `true|false` | (No reference in code???)                                                  | `true`  |
+| `format.hist_election_rows`           | boolean | `true|false` | (No reference in code???)                                                  | `true`  |
+
```

Comment:
yeah I don't think those ever get used, I assume it was just for human readable understanding at some point?